packages/webapp/src/components/Translate.tsx

```diff
@@ -598,7 +743,7 @@ export default observer(function Translate() {
 						<section>
 							<div className="filterGroup">
 								<div>
-									<label htmlFor={"availDropdown"}>Translate from:</label>
+									<label htmlFor={'availDropdown'}>Translate from:</label>
```

Comment:
in consistant with strings right above this. also strings shouldn't be in brackets